datahub-ranger-plugin/README.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Configuring Apache Ranger"
+hide_title: true
+---
+# Configuring Apache Ranger
+Datahub Apache Ranger Plugin configuration is consist of below configuration sections 
+1. Configuration at Apache Ranger Deployment 
+2. Configuration at Datahub Deployment
+
+The configuration sections are tested on [Privacera Platform](https://privacera.com/) v6.3.0.1.
+
+# Assumption 
+- Apache Ranger and DataHub are configured with same IDP
+- Apache Ranger service is running on HTTP
+
+# Configuration 
+## Configuration at Apache Ranger Deployment 
+1. Download the datahub-ranger-plugin from [Maven](https://mvnrepository.com/artifact/io.acryl/datahub-ranger-plugin)
+2. Create  *datahub*  directory in each *Privacera Ranger* container 
+```bash
+docker exec privacera_ranger_1 mkdir ews/webapp/WEB-INF/classes/ranger-plugins/datahub
+```
+3. Copy the datahub-ranger-plugin jar into *Privacera Ranger* container
+```bash 
+docker cp datahub-ranger-plugin-<version>.jar privacera_ranger_1:/opt/ranger/ranger-2.1.0-admin/ews/webapp/WEB-INF/classes/ranger-plugins/datahub/
+```
+4. Download the [service definition](../datahub-ranger-plugin/conf/servicedef.json). This service definition is the ranger service definition JSON file for datahub-ranger-plugin-<version\>.jar
+5. Execute below curl command to register the service definition <br /> 
```

Comment:
"Register the service definition: "

Then show _how_..

For each step, start with a clear few words about the step.

Reply:
Done